provider.tf

```diff
@@ -0,0 +1,4 @@
+provider "aws" {
+  profile = "reut"
+  region  = var.aws_region
+}
```

Comment:
I moved the provider to its own file. It is not necessarily related to the vpc.tf file